### ISSUE TITLE
Building android does not find task in gradlew

### ISSUE THREAD

**Issue by AstralisSomnium** [Shippable/support#2998]
I'm trying to build my android project from github:
https://github.com/AstralisSomnium/Test

I always get following exception on "./gradlew assembleDebug" or "./gradlew  assembleRealse":
Task 'assembleDebug' not found in root project 'Test'.

 My shippable.yml looks like this:
language: java
jdk:
  - oraclejdk8
  
build:         
  ci:
    - chmod +x gradlew
    - ./gradlew clean
    - ./gradlew assembleDebug
    
integrations:
    notifications:
        - integrationName: SendEmail
          type: email
          recipients:
            - gruberpatrickit@gmail.com
          on_success: always
          on_failure: always

This is the link to the failed build:
https://app.shippable.com/runs/57e969bfdd44c60e00dcc998/1/console

**Comment by a-murphy** [comment]
As far as I can tell, your build isn't finding the gradle file in `app`.  It looks like you're missing a `settings.gradle` file that would tell it where to look.  Could you try adding one as documented [here](https://developer.android.com/studio/build/index.html#settings-file)?

**Comment by AstralisSomnium** [comment]
Thank you for the answer, I will give it a try and give feedback!
But could it also be that this issue is caused by not have installed the Android SDK on this docker image? Because I use the default docker image from shippable, maybe I have to use a custom one?

**Comment by a-murphy** [comment]
The default Shippable images do not have the Android SDK pre-installed, but it isn't related to the error above.  You can either install it in the `ci` section of your shippable.yml or use a custom image.  Documentation on specifying custom images is available [here](http://docs.shippable.com/ci/shippableyml/#pre_ci_boot).  Ubuntu-based images usually work best.

**Comment by AstralisSomnium** [comment]
Thanks the issue where the task was not found was solved by adding the settings.gradle!
Now I faced the issue with the not installed android SDK, but since I do not have any experience with docker so far, I will try the solution and install the SDK before the build happens.
The prefered way would be to use a custom docker image? Because I can imagine that it takes a long time to download and install the android SDK in EACH build..
